src/index.html

```diff
@@ -17,11 +18,408 @@
     />
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="./index.scss"
     />
   </head>
+  <!-- endregion -->
 
   <body>
-    <h1>Catalog</h1>
+    <!-- #region header -->
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo Logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              href="#"
+              class="is-active nav__link"
+            >
+              apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              data-qa="nav-hover"
+              class="nav__link"
+            >
+              laptops & computers
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              gadgets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              tablets
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              photo
+            </a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+            >
+              video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <!-- #endregion -->
+
+    <div class="main">
+      <!-- #region card -->
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          class="card__image"
+          src="../src/images/imac.jpeg"
+          alt="imac"
+        />
+        <h1 class="card__title">
```

Comment:
```suggestion
        <h2 class="card__title">
```
h1 is a title of the entire page

Reply:
<img width="661" alt="image" src="https://github.com/user-attachments/assets/abdf91e2-8b50-4ff5-8627-2134b7b254c7">